scripts/process-templates.js

```diff
@@ -0,0 +1,47 @@
+
+const program = require('commander')
+const nunjucks = require('nunjucks')
+const glob = require('glob')
+const fs = require('fs')
+const path = require('path')
+
+program.version('0.0.1')
+program.option('-c, --child-chain-id <child-chain-id>', 'Child chain id', '15001')
```

Comment:
lets keep default chain id as `80001` since that is mumbai chain id

Reply:
No. We should keep our local default.